app/components/sidebars/drawer_layout.js

```diff
@@ -45,6 +45,7 @@ export type PropType = {
     renderNavigationView: () => any,
     statusBarBackgroundColor?: string,
     useNativeAnimations?: boolean,
+    isMain: boolean,
```

Comment:
do you really need `isMain` when you already have the `drawerPosition` ? can you elaborate on your thought process here?

Reply:
You know... no 😂 I let the RTL trip me up there, but since handling RTL _after_ the `drawerPosition` is passed in, that will of course work.

Originally I put in a prop called `displaceContent`, but decided to just put in `isMain` so that it was explicitly clear to the next developer that the conditions that apply to `isMain` are solely because it is the main sidebar, and not any other one. This is otherwise just inferred from `drawerPosition` and knowing there are only two `DrawerLayout` components in use. Less props, possibly more confusing 🤷‍♂️

Personally I think using "left" is cleaner now - want me to update?

Reply:
I see your point and it makes total sense, as of now we do not support RTL and who knows when will that happen, so yes, maybe lets go with `left` for now.